roles/servicetelemetry/templates/manifest_smartgateway_events.j2

```diff
@@ -11,8 +11,8 @@ spec:
   debug: false
   elasticUrl: 'https://elasticsearch-es-http.{{ meta.namespace }}.svc.cluster.local:9200'
   useBasicAuth: true
-  elasticUser: '{{ elastic_user }}'
-  elasticPass: '{{ elastic_pass }}'
+  elasticUser: '{{ elastic_user | default('elastic') }}'
```

Comment:
For note why this isn't exposed, it's because these are basically done via internal lookup to populate, so I thought exposing them are parameters was not appropriate. Keeps the logic a bit simpler too.